examples/scale-to-zero-with-keda/assets/scaling-scenario.drawio

```diff
@@ -0,0 +1,248 @@
+<mxfile host="app.diagrams.net" modified="2024-01-08T12:51:53.735Z" agent="Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/120.0.0.0 Safari/537.36" etag="ctpqmkFrrgU9wYFfphru" version="22.1.16" type="device">
```

Comment:
This looks weird :D 

<img width="206" alt="image" src="https://github.com/kyma-project/keda-manager/assets/76950046/0ab18d8f-71db-4b12-8f25-a62f539b2b21">

Reply:
isn't this an international symbol for api exposure? ;)

Reply:
Now it looks weird in a different way!

<img width="148" alt="image" src="https://github.com/kyma-project/keda-manager/assets/76950046/d53e9e48-bf5f-4184-ba29-ca2284051be3">

([Use white background instead of transparent](https://github.com/kyma-project/community/blob/main/docs/guidelines/content-guidelines/09-diagrams.md#background))